waiter/integration/waiter/request_timeout_test.clj

```diff
@@ -180,12 +180,13 @@
 (deftest ^:parallel ^:integration-fast test-grace-period-with-tokens
   (testing-using-waiter-url
     (let [grace-period (t/minutes 2)
+          startup-delay-ms (-> grace-period t/in-millis (* 0.75) long)
           token (rand-name)]
       (try
         (log/info "Creating token for" token)
         (let [{:keys [status body]}
               (post-token waiter-url {:cmd (kitchen-cmd (str "--port $PORT0 --start-up-sleep-ms "
-                                                             (t/in-millis grace-period)))
+                                                             startup-delay-ms))
```

Comment:
Can we include an assertion on the canary request that it takes at least `startup-delay-ms` ms to come back with a response?

Reply:
Done.